guides/advanced_guides/configuration.md

```diff
@@ -220,6 +221,8 @@ Depending on the OS, it is either the size that will be allocated on launch or t
 
 [To know more about storage in MeiliSearch look at this guide](/resources/about_storage.md)
 
+ By default, This size is in bytes but it can be represented in different units (bytes, kilobytes, megabytes, etc..), more information [here](#available-size-units).
+
 **Default value**: `107374182400` (100 GiB)
 
 ### Max UDB Size
```

Comment:
Will this update also affect Max UDB size?

Reply:
I understood yes from here :+1: 
 "And that would be great to also offer this update to the UDB size and MDB size configuration."

Reply:
If you confirm me this... I rewrite the doc for the aproval.

Reply:
Yes, it's confirmed. Thank you :)